src/types/onyx/SearchResults.ts

```diff
@@ -48,6 +48,8 @@ type SearchTransaction = {
     policyID: string;
     transactionThreadReportID: string;
     shouldShowMerchant: boolean;
+    shouldShowCategory: boolean;
+    shouldShowTag: boolean;
```

Comment:
I agree that we should have 2 separate types one for API data and one for view data. 

>In general I'm not the biggest fan of the fields addition in getSections fn, but if they work and you like it then we can keep it.

Do you have a alternative proposal for this?

Reply:
FYI I talked with Wojtek about this and we didn't find a better solution, because you need the whole set of `data` which I missed the first time I looked at this.

Any way of passing all items to `TransactionListItem` would be a bad idea I believe, so we're keeping this here but with improved types